src/vimeo/players/shared/CloseButton.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+import WDSVariables from '@fandom-frontend/design-system/dist/variables.json';
+import IconCrossTiny from '@fandom-frontend/react-common/dist/icons/IconCrossTiny';
+import { VimeoArticleVideoPlayerTrackingProps } from 'vimeo/types';
+
+const CloseWrapper = styled.div<VimeoArticleVideoPlayerTrackingProps>`
+	cursor: pointer;
+	pointer-events: initial;
+	height: ${(props) => (props.deviceType === 'desktop' ? '24px' : '36px')};
+	width: ${(props) => (props.deviceType === 'desktop' ? '24px' : '36px')};
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	opacity: 0.98;
+	position: absolute;
+	right: -3px;
+	right: ${(props) => (props.deviceType === 'desktop' ? '-3px' : '-6px')};
+	top: -3px;
+	z-index: ${Number(WDSVariables.z7) + 1};
+`;
+
+const CrossIcon = styled(IconCrossTiny)`
+	fill: ${WDSVariables.wdsColorWhite};
+`;
+
+interface CloseButtonProps extends VimeoArticleVideoPlayerTrackingProps {
+	dismiss: () => void;
+}
+
+const CloseButton: React.FC<CloseButtonProps> = ({ dismiss, deviceType }) => {
+	const onClickClose = () => {
+		dismiss();
+	};
```

Comment:
I think the intermediate `onClickClose` function isn't necessary and you could pass `dismiss` directly to the `CloseWrapper` component.